src/main/kotlin/no/skatteetaten/aurora/gobo/security/PsatSecretReader.kt

```diff
@@ -0,0 +1,68 @@
+package no.skatteetaten.aurora.gobo.security
+
+import mu.KotlinLogging
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+import org.springframework.stereotype.Component
+import java.io.File
+import java.io.IOException
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * Component for reading the shared secret used for authentication with psat token. You may specify the shared secret directly using
+ * the aurora.psat.values.<audience> property, or specify a directory containing files with secrets for psat with the aurora.psat.location property.
+ *
+ * Examples:
+ * aurora.psat.location: /dir/to/psat
+ * aurora.psat.values.spotless: <token-value>
+ */
+@Component
+class PsatSecretReader(
```

Comment:
Kan vi bruke kubernetes-client biblioteket her i stedet for å lage dette helt på nytt?
https://github.com/Skatteetaten/kubernetes-reactor-coroutines-client/blob/master/src/main/kotlin/no/skatteetaten/aurora/kubernetes/config/KubernetesClientConfig.kt#L91

Reply:
jeg har endret til å bruke WebClient med PsatTokenFetcher fra kubernetes-client biblioteket ref. ApplicationConfig